### DISCUSSION TITLE
Filters Formatting

### DISCUSSION THREAD

**Discussion by ebussinetz** [liedekef/events-made-easy#651] [Q&A] [answered]
The new #_FILTER_YEARS{40}{1} is great, thank you!

I do have a suggestion for improvement:
The selectors have different HTML structures and that's why it's difficult to make them uniform. It would be perfect if they were all structured uniformly in HTML.
Some have an arrow, e.g. #_FILTER_CATS, which is in its own span tag and some, like #_FILTER_YEARS, have no arrow (and no span for it either).
And although it's not that important, it would be nice if you could specify the sorting of the years, e.g. the oldest first or the newest first (in the selector).
It would also be good if I could specify the oldest year in the past instead of the number of years in the past. Otherwise you have to change the number every year ;-)
You can see it in action here:
https://www.spiritualchor.de/termine/vergangene-auftritte/

Thank you and kind regards
Wolfgang

**Comment by liedekef** [comment, marked as answer]
The dropdown arrow for the year is being hidden by your theme, not by EME. The selects for the category and city look different because a jquery (by EME) is transforming them into searchable items (which is not needed for years).
Concerning the re-order: it would be possible, but requires coding (either a new placeholder or extra option to the current ones). For now, since this is not a commonly asked something: this is on hold unless funding is provided. A workaround is to use own jquery to change the order of a field (which is like a 5-minute job, see for example: https://stackoverflow.com/questions/23162208/how-to-reorder-a-dropdown-list-jquery ).
